### ISSUE TITLE
Cannot retrieve any DNS SRV adress after typing "."

### ISSUE THREAD

**Issue by Nan0fm** [MiniDNS/minidns#91]
I've using this lib for retrieving SRV records on Android app.

Steps
1. Get SRV records from "_mydomain._tcp.dev.local"
 ResolverResult<SRV> result -> responseCode = NO_ERROR ------- works perfectly
2. Try to get "_mydomain._tcp.dev.local."
ResolverResult<SRV> result -> responseCode = NX_DOMAIN -------- okay, works as expected
3. Get SRV records from "_mydomain._tcp.dev.local"
 ResolverResult<SRV> result -> responseCode = NX_DOMAIN ----------- what? why? 
The result of the following attempts will be the same until I restart the app

Why I retrieve responseCode = NX_DOMAIN in third step?

**Comment by Flowdalic** [comment]
I'm sorry but your description is hard to follow. It would help tremendously if you showed some *short* and relevant code snippets. Or provide a minimal example which attempts to reproduce the behavior.

Given that my interpretation of your description is correct, my first intuition would be we are possibly dealing which erroneous caching behavior by MiniDNS here.

**Comment by Nan0fm** [comment]
@Flowdalic thanks for quick response :)
 I think you are right,  problem is in caching, but how I can clear caching for miniDNS?

**Comment by Nan0fm** [comment]
@Flowdalic 
If this will help you, this throws when I'm trying to query adress with "." in the end of adress and all time after this
![image](https://user-images.githubusercontent.com/30218236/45678056-4f716c80-bb3e-11e8-9832-3bf26fcb0763.png)

**Comment by Nan0fm** [comment]
@Flowdalic 
Hi :) Is any updates for this issue?

**Comment by Flowdalic** [comment]
I'm unable to reproduce this with the latest MiniDNS master. That is

```java
    public void testClient() throws IOException {
        DnsClient dnsClient = new DnsClient(new LruCache(1024));

        DnsQueryResult result1 = dnsClient.query("geekplace.eu", TYPE.A);
        
        DnsQueryResult result2 = dnsClient.query("geekplace.eu.", TYPE.A);
        
        DnsQueryResult result3 = dnsClient.query("geekplace.eu", TYPE.A);

    }
```

works as expected. Please provide a minimal code snipped which can be used to reproduce your issue and state the MiniDNS version you are using. I'm closing this until the code to reproduce this issue has been provided.
